src/components/StoreRevokeBlock.js

```diff
@@ -16,43 +19,62 @@ class StoreRevokeBlock extends Component {
 
   onHashChange(event) {
     this.setState({
-      certificateHash: event.target.value
+      certificateHash: event.target.value,
+      certificateHashIsValid: isValidCertificateHash(event.target.value)
     });
   }
 
   onRevokeClick() {
     const { adminAddress, storeAddress, handleCertificateRevoke } = this.props;
+    const { certificateHash } = this.state;
 
-    const yes = window.confirm("Are you sure you want to revoke this hash?"); // eslint-disable-line
-
-    if (yes) {
-      handleCertificateRevoke({
-        storeAddress,
-        fromAddress: adminAddress,
-        certificateHash: this.state.certificateHash
-      });
+    this.setState({
+      certificateHashIsValid: isValidCertificateHash(certificateHash)
+    });
+    if (isValidCertificateHash(certificateHash)) {
+      // eslint-disable-next-line no-alert
+      const yes = window.confirm("Are you sure you want to revoke this hash?");
+      if (yes) {
+        handleCertificateRevoke({
+          storeAddress,
+          fromAddress: adminAddress,
+          certificateHash
+        });
+      }
     }
   }
 
   render() {
+    const { certificateHash, certificateHashIsValid } = this.state;
     const { revokedTx, networkId } = this.props;
+    const certificateHashMessage = certificateHashIsValid
+      ? ""
+      : "Merkle Root Hash is not valid.";
```

Comment:
```suggestion
      : "Merkle root/target hash is not valid.";
```

Not sure if this will help remind users that they can either revoke the entire tree or just the target hash.